app/shared/db/props.server.ts

```diff
@@ -20,9 +20,11 @@ export const loadByProject = async (
 };
 
 export const loadByTreeId = async (treeId: Tree["id"]) => {
-  return await prisma.instanceProps.findMany({
+  const trees = await prisma.instanceProps.findMany({
```

Comment:
hmm these are not trees, those are instancePropsEntries or something

Reply:
Truuum fixed